runhouse/main.py

```diff
@@ -98,6 +102,116 @@ def ssh(cluster_name: str, up: bool = typer.Option(False, help="Start the cluste
+
+    # print cluster config
+    first_info_to_print = ["den_auth", "server_connection_type", "server_port"]
+
+    for info in config:
+        if info in first_info_to_print:
+            console.print(f"\u2022 {info}: {config[info]}")
+    first_info_to_print.append("name")
+
+    console.print("\u2022 backend config:")
+    for info in config:
+        if info not in first_info_to_print:
+            console.print(f"\t\u2022 {info}: {config[info]}")
+
+    # print the environments in the cluster, and the resources associated with each  environment.
+    envs_in_cluster_headline = "Serving 🍦 :"
+    console.print(envs_in_cluster_headline, style="bold")
+
+    if len(envs) == 0:
+        console.print("This cluster has no environment nor resources.")
+
+    for env_name in envs:
+        resources_in_env = envs[env_name]
+        if len(resources_in_env) == 0:
+            console.print(f"{env_name} (Env):", style="italic underline")
+            console.print("This environment has no resources.")
+
+        else:
+            current_env = [
+                resource
+                for resource in resources_in_env
+                if resource["name"] == env_name
+            ]
+
+            # sometimes the env itself is not a resource (key) inside the env's servlet.
+            if len(current_env) == 0:
+                env_name_txt = f"{env_name} (Env):"
+            else:
+                current_env = current_env[0]
+                env_name_txt = (
+                    f"{current_env['name']} ({current_env['resource_type']}):"
+                )
+
+            console.print(
+                env_name_txt,
+                style="italic underline",
+            )
+
+            resources_in_env = [
+                resource for resource in resources_in_env if resource is not current_env
+            ]
+
+            for resource in resources_in_env:
+                resource_name = resource["name"]
+                resource_type = resource["resource_type"]
+                console.print(f"\u2022{resource_name} ({resource_type})")
+
+    return config
+
+
+@app.command()
+def status(cluster_name: str = typer.Argument(None, help="Name of cluster to check")):
+    """Get the config info about your runhouse cluster(s), as well as the environments setup of the cluster(s)"""
+
+    cluster_or_local = rh.here
+
+    if cluster_or_local == "file":
+        if cluster_name is None:
+            console.print("Missing argument CLUSTER_NAME")
+            return
+        else:
+            try:
+                current_cluster = rh.cluster(name=cluster_name)
+                if not current_cluster.is_up():
+                    console.print(f"A cluster called {cluster_name} is not up")
+                    return
+                current_cluster.connect_server_client()
+                config = current_cluster.client.request(
+                    endpoint="/status", req_type="get"
+                )
+            except Exception:
```

Comment:
I think we should actually break these out into two separate exceptions to be more descriptive: 
`ValueError`: Cluster not found
`ConnectionError`: Cluster exists but daemon not running properly